packages/server/graphql/mutations/endRetrospective.ts

```diff
@@ -93,6 +94,7 @@ export default {
       return standardError(new Error('Cannot find facilitator stage'), {userId: viewerId})
     }
     const {stage} = currentStageRes
+    await handleCompletedStage(stage, meeting, dataLoader)
```

Comment:
When we end the meeting, `handleCompletedStage` isn't called from `navigateMeeting` so the `discussionSummary` isn't generated. To fix, I've added this here in `endRetrospective`